### DISCUSSION TITLE
Fabrication v2: Initial commit prep

### DISCUSSION THREAD

**Discussion by MrTJP** [MrTJP/ProjectRed#1728] [Announcements]
Hello!

It's time for another update on the Fabrication module. This is well overdue. I'll try and do these more regularly. I'm getting ready to post an initial commit for the module, and then do an early beta release about a week later. 

I created one of those nifty Github project boards to keep track of stuff left to do: [Fabrication v2](https://github.com/users/MrTJP/projects/4)

Tons of progress has been made on the UI, but I feel like I keep trying to refine things rather than focusing on having a simple initial implementation that works well enough. I am dropping all non-essential work until I have a basic minimum viable product that I can release. 

So what will be included in the initial Beta release?
1. Basic implementation of the UI. It will be at about feature parity with original Fabrication release in 1.12, with multi-layered designs being possible.
2. No gameplay elements or recipes. There will be a single Workbench block, a blueprint item, and a gate block. Blueprint will be directly convertible into a gate. This is enough to do basic testing of your design in-world in creative mode. Next release will focus on bringing things into Survival with proper recipes, progression, etc.
3.  Limited number of gates available initially. Likely will have almost all of the simple stuff but no Bundled or Array cell gates (Bundled gates are now possible though 😀)

<img width="1035" alt="Screen Shot 2022-05-11 at 12 00 57 PM" src="https://user-images.githubusercontent.com/1957405/167903650-ea93d145-72fe-4ba9-9cc9-a65cc78fd517.png">

**Comment by Rayzeq** [comment]
This look really cool!!! I can't wait for the final version to be released !

**Comment by bauerbach** [comment]
How do you get access to the releases?
